### ISSUE TITLE
build.sh incorrectly sets PATH for downloaded Go toolchain

### ISSUE THREAD

**Issue by MFAshby** [foxcpp/maddy#201]
# Describe the bug

Using the build.sh script [as described in the documentation](https://foxcpp.dev/maddy/tutorials/setting-up/) doesn't work

# Steps to reproduce
run `curl 'https://foxcpp.dev/maddy/build.sh' | bash`

# Log files
```
root@ubuntu-s-1vcpu-1gb-sgp1-01:~# curl 'https://foxcpp.dev/maddy/build.sh' | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 15643  100 15643    0     0  23209      0 --:--:-- --:--:-- --:--:-- 23174
                                _|        _|
_|_|_|  _|_|      _|_|_|    _|_|_|    _|_|_|  _|    _|
_|    _|    _|  _|    _|  _|    _|  _|    _|  _|    _|
_|    _|    _|  _|    _|  _|    _|  _|    _|  _|    _|
_|    _|    _|    _|_|_|    _|_|_|    _|_|_|    _|_|_|
                                                    _|
 All-in-one composable mail server.             _|_|

--- Downloading Go 1.13.4 toolchain...
--- Using downloaded Go toolchain (1.13.4, go1.13.4/bin/go).
main: line 243: go: command not found
```

# Configuration file

N/A

# Environment information

Start a new droplet on DigitalOcean (ubuntu 18.04.3 LTS)

* maddy version: 0.1

**Comment by foxcpp** [comment]
Can you try again?

**Comment by MFAshby** [comment]
Thanks! Different to before -- looks like it managed to execute `go` but has some problem with packages. See attached [log.txt](https://github.com/foxcpp/maddy/files/4240391/log.txt)

[Manual install](https://foxcpp.dev/maddy/tutorials/manual-installation) worked ok

**Comment by foxcpp** [comment]
Okay, I see. Go 1.13.4 downloaded by script reports missing go.mod as GOMOD="" while Go 1.13.8 (version I used when writing build.sh) sets GOMOD="/dev/null". Therefore script checks for the latter and skips source download thinking it is already here.

EDIT: Ah! Not that, I had `GO111MODULE=on` set globally on my system and this is what changes GOMOD value.

**Comment by foxcpp** [comment]
Should be fixed now (verified on Ubuntu 18.04 container), forgot to close the issue.

**Comment by MFAshby** [comment]
Confirmed it's fixed, thankyou.
